preliz/tests/test_beta_mode.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import numpy as np
+from preliz import beta_mode
+
+
+@pytest.mark.parametrize(
+    "lower, upper, mode, mass, expected_alpha, expected_beta",
+    [
+        (0.25, 0.75, 0.5, 0.9, 4.94, 4.94),  # Example test case
+        # Add more test cases here
```

Comment:
remove these comments. 

Are you going to add more tests here? If not there is not need to use "@pytest.mark.parametrize" just write the values inside the function

Reply:
i have removed the comments and pytest parameterise in the latest commit.